source/products/mobiliar/flixbus/rest-api.rst

```diff
@@ -0,0 +1,188 @@
+       -H 'Accept: application/vnd.kasko.v2+json' \
+       -d touchpoint_id=tp_21716734231da536b06e0a624719f \
+       -d item_id=item_e24329e6750469ee0bd5a92c65a \
+       -d subscription_plan_id=sp_8f603535e76eb895124c9d2785c1b \
+       -d data='{"departure_location":"zurich","arrival_location":"grenoble","departure_date":"2019-04-11","departure_time":"09:00","delay":0,"payout_amount":35}'
+
+.. _QuoteResponse:
+
+Sample response
+~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+    {
+        "token": "<QUOTE TOKEN>",
+        "gross_payment_amount": 307,
+        "extra_data": {
+            "gross_premium": 307,
+            "premium_tax": 15,
+            "net_premium": 292,
+            "tax_rate": 0.05,
+            "end_date": "2019-04-11 09:00",
+            "start_date": "2019-04-11 09:01"
+        }
+    }
+
+.. _Offer:
+
+Create an offer (unpaid policy)
+----------------------------
+
+This request stores policy holder information that is related to offer. Following information can be stored in offer:
+
+.. csv-table::
+   :header: "Name", "Type", "Description", "Example Value"
+   :widths: 35, 20, 75, 20
+
+   "phone",                           "string",   "Free text string up to 255 characters.",   "+417304200"
+
+Example Request
+~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+	curl -X POST \
+	  'https://api.kasko.io/policies' \
+	  -u sk_test_SECRET_KEY: \
+	  -H 'Accept: application/vnd.kasko.v2+json' \
+	  -H 'Content-Type: application/json' \
+	  -d '{
+          "data": {
+                "phone":"+11111111"
+          },
+          "quote_token":"<QUOTE TOKEN>",
+          "first_name": "Test",
+          "last_name": "Person",
+          "email": "test@kasko.io",
+          "language": "en"
+      }'
+
+NOTE. You should use ``<QUOTE TOKEN>`` value from QuoteResponse_.
+
+.. _OfferResponse:
+
+Sample response
+~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+    {
+        "id": "<POLICY ID>",
+        "insurer_policy_id": "TEST-MOB-34L3638J876",
+        "payment_token": "<PAYMENT TOKEN>",
+        "_links": {
+            "_self": {
+                "href": "https://api.kasko.io/policies/<POLICY ID>"
+            }
+        }
+    }
+
+.. _Payment:
+
+Convert offer to policy (payment)
+---------------------------------
+
+To create a policy you should convert offer to policy. In other words - make payment for the offer.
+This can be done by making following request:
+
+.. csv-table::
+   :header: "Parameter", "Required", "Type", "Description"
+   :widths: 20, 20, 20, 80
+
+   "token",     "yes", "``string``", "The ``<PAYMENT TOKEN>`` returned by OfferResponse_."
+   "policy_id", "yes", "``string``", "The 33 character long ``<POLICY ID>`` returned by OfferResponse_."
+   "method",    "yes", "``string``", "Payment method ``invoice``."
+   "provider",  "yes", "``string``", "Payment provider ``invoice``."
+   "metadata.iban",  "yes", "``string``", "``DE89370400440532013000``"
+   "metadata.bic",  "yes", "``string``", "``BYLADEM1001``"
```

Comment:
Is this really the case? Are we collecting IBAN & bic for this product? AFAIK it should be a payment request with no metadata.